plugin.video.retrospect/addon.xml

```diff
@@ -145,4 +151,4 @@ Mainly a release to update the Retrospect license from CC BY-NC-ND to CC BY-NC-S
             <screenshot>resources/media/retrospect03.jpg</screenshot>
         </assets>
     </extension>
-</addon>
\ No newline at end of file
+</addon>
```

Comment:
I've tried numerous times to remove the empty line at the end, both by editing online and locally (before making the PR). Github won't let me!

Reply:
@Snaah Don't update your PR via GitHub. Do it locally and then push the changes.